Settings/SettingsView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SettingsView.swift
+//  Shapify
+
+import UIKit
+
+final class SettingsView: UIView {
+    
+    private let infoLabel = UILabel().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.font = .appFont(ofSize: 36, isBold: true)
+        $0.textColor = .black
+        $0.textAlignment = .center
+        $0.text = "Settings"
+        $0.numberOfLines = 0
+    }
+    
+    private let profilePicture = UIImageView().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        let urlString = ProfileManager.shared.profile?.imageURL
+        let url = URL(string: urlString!)!
+        let data = try? Data(contentsOf: url)
+            if let data = data {
+                $0.image = UIImage(data: data)
+            } else {
+                $0.image = UIImage(named: "customerIcon")
+            }
+        $0.setWidth(0.5 * .screenWidth)
+        $0.layer.cornerRadius = 0.25 * .screenWidth
+        $0.clipsToBounds = true
+    }
+    
+    private let usernameLabel = UILabel().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.font = .appFont(ofSize: 28)
+        $0.textColor = .black
+        $0.textAlignment = .center
+        $0.text = ProfileManager.shared.profile?.username ?? "username"
+        $0.adjustsFontSizeToFitWidth = true
+    }
+    
+    private let emailLabel = UILabel().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.font = .appFont(ofSize: 28)
+        $0.textColor = .black
+        $0.textAlignment = .center
+        $0.text = ProfileManager.shared.profile?.email ?? "e-mail address"
+        $0.adjustsFontSizeToFitWidth = true
+    }
+    
+    let logOutButton = UIButton().then {
+        $0.setUpRoundedButton(title: "Log out")
+    }
+    
+    public init() {
+        super.init(frame: .zero)
+        setUp()
+    }
+    
+    @available(*, unavailable)
+    
+    required public init?(coder: NSCoder) { nil }
```

Comment:
I suggest inheriting after BaseView class, which then makes those lines unneeded. If you decide to change it, you should call only super.init() instead of super.init(frame: .zero).